datafusion/core/tests/sqllogictests/test_files/union.slt

```diff
@@ -76,3 +74,121 @@ ORDER BY name
 ----
 3 Alice
 3 John
+
+# union all
+query T
```

Comment:
```suggestion
query I
```

There is currently no type check, but if this change ever lands https://github.com/risinglightdb/sqllogictest-rs/issues/36#issuecomment-1341031082, then it would be fewer changes during the sqllogictest-rs version update.